setup.py

```diff
@@ -9,8 +9,8 @@
     requirements = f.read().splitlines()	
 
 setup(	
-    name="dspy-ai",	
-    version="2.4.9",	
+    name="{{PACKAGE_NAME_PLACEHOLDER}}",
```

Comment:
are they actual place holder or they automatically will be filled up?

Reply:
oh nvm, I think they are filled up with 
```      - name: Update package name in pyproject.toml
        run: sed -i "s/{{PACKAGE_NAME_PLACEHOLDER}}/dspy-ai-test/g" pyproject.toml```

Reply:
Yes they are automatically filled in by the version as set by the tag (or with the version + an auto incremented identifier in case multiple pushes are made to test pypi)